src/main/java/org/wildfly/security/sasl/util/AbstractSaslParticipant.java

```diff
@@ -67,11 +67,12 @@
      * @param serverName the server name
      * @param callbackHandler the callback handler
      */
-    protected AbstractSaslParticipant(final String mechanismName, final String protocol, final String serverName, final CallbackHandler callbackHandler) {
+    protected AbstractSaslParticipant(final String mechanismName, final String protocol, final String serverName, final CallbackHandler callbackHandler, ElytronMessages log) {
```

Comment:
Again this class is public in a package flagged as being public API so we do need to retain compatibility.